perun-base/src/main/resources/perun-roles.yml

```
@@ -4569,6 +4569,23 @@ perun_policies:
     include_policies:
       - default_policy
 
+  getEnrichedBansForResource_int_List<String>_policy:
+    policy_roles:
+      - PERUNOBSERVER:
+      - FACILITYADMIN: Facility
+      - FACILITYOBSERVER: Facility
+    include_policies:
+      - default_policy
+
+  getEnrichedBansForUserOnResource_int_List<String>_policy:
```

I would suggest this change since the original may be misleading.  In my opinion it sounds like it would return bans for one resource rather than bans for all resources.
```suggestion
  getEnrichedResourceBansForUser_int_List<String>_policy:
```